docs/reference-guides/data/data-core-edit-post.md

```diff
@@ -81,6 +81,15 @@ _Returns_
 
 -   `string`: Editing mode.
 
+### getHiddenBlockTypes
+
+Registry selector that gets the hidden block types from the preferences
+store.
```

Comment:
I think the descriptions for the selector and two actions are a little too descriptive of the implementation. A plugin author probably doesn't care how it's implemented, just what happens when e.g. `hideBlockTypes` is dispatched, i.e. the block is hidden from the inserters.